src/lib/AnnotationControls.ts

```diff
@@ -42,13 +103,15 @@ export default class AnnotationControls {
      * @return {void}
      */
     private handleRegionClick = (onRegionClick: RegionHandler) => (event: MouseEvent): void => {
-        const regionButtonElement = event.target as HTMLButtonElement;
+        const regionButtonElement = this.controlsElement.querySelector(`.${CLASS_REGION_BUTTON}`);
 
-        this.isRegionActive = !this.isRegionActive;
-        if (this.isRegionActive) {
-            regionButtonElement.classList.add(CLASS_BUTTON_ACTIVE);
-        } else {
-            regionButtonElement.classList.remove(CLASS_BUTTON_ACTIVE);
+        if (regionButtonElement) {
+            this.isRegionActive = !this.isRegionActive;
+            if (this.isRegionActive) {
+                regionButtonElement.classList.add(CLASS_BUTTON_ACTIVE);
+            } else {
+                regionButtonElement.classList.remove(CLASS_BUTTON_ACTIVE);
+            }
```

Comment:
Once the toggleAnnotationMode api is setup with the corresponding event, we may want to consider refactoring this to respond to the event instead so we can remove the local state maintained by AnnotationControls